core/src/reporter/utils.ts

```diff
@@ -179,9 +179,15 @@ export async function sendTransactionDelegatedFee({
   let response
   try {
     response = (
-      await axios.post(endpoint, {
-        ...transactionData
-      })
+      await axios.post(
+        endpoint,
+        {
+          ...transactionData
+        },
+        {
+          timeout: 3000
```

Comment:
How about making the timeout updatable through environment variable defined at `settings.ts`?